Flutter/datagrid/selection.md

```diff
@@ -1613,5 +1613,5 @@ Widget build(BuildContext context) {
 
 ![flutter datagrid shows customized current cell](images/selection/flutter-datagrid-customized-currentcell.png)
 
-
-
+>**NOTE**  
```

Comment:
You can change the current row border's color by the `currentCellStyle` property of `SfDataGridThemeData` in `SfDataGridTheme`. The current row border is shown when `navigationMode` is `row` to navigate between rows.

Reply:
Modified